pkgs/development/libraries/json-c/default.nix

```diff
@@ -7,6 +7,14 @@ stdenv.mkDerivation rec {
     sha256 = "0ch1v18wk703bpbyzj7h1mkwvsw4rw4qdwvgykscypvqq10678ll";
   };
 
+  patches = [
```

Comment:
can you link to https://nvd.nist.gov/vuln/detail/CVE-2020-12762?

Reply:
Added link.